lib/shopify_app/scripttags_manager.rb

```diff
@@ -44,12 +37,6 @@ def is_required_scripttag?(scripttag)
       required_scripttags.map{ |w| w[:src] }.include? scripttag.src
     end
 
-    def with_shopify_session
-      ShopifyAPI::Session.temp(@shop_name, @token) do
-        yield
-      end
-    end
-
     def create_scripttag(attributes)
       attributes.reverse_merge!(format: 'json')
       scripttag = ShopifyAPI::ScriptTag.create(attributes)
```

Comment:
In the PR where we expose this method we should add specific errors that are raised when there is no ShopifyAPI session set.